src/app.service.ts

```diff
@@ -1,8 +1,12 @@
 import { Injectable } from '@nestjs/common';
+import { stringConstants } from './utils/string.constant';
 
 @Injectable()
 export class AppService {
-  getHello(): string {
-    return 'Hello World!';
+  getHello(): {app_version: string, env: string} {
```

Comment:
can u rename this function? instead of getHello you can use something like "getAppInfo" or getServiceInfo